bin/wmcore-db-init

```diff
@@ -121,7 +128,18 @@ def create(config):
     if hasattr(config.CoreDatabase, "indexspaceName"):
         params["tablespace_index"] = config.CoreDatabase.indexspaceName
 
-    wmInit.setSchema(modules, params = params) 
+    wmInit.setSchema(modules, params = params)
+    return
+
+def destroy(config):
+    params = {}
+
+    if hasattr(config.CoreDatabase, "tablespaceName"):
+        params["tablespace_table"] = config.CoreDatabase.tablespaceName
+    if hasattr(config.CoreDatabase, "indexspaceName"):
+        params["tablespace_index"] = config.CoreDatabase.indexspaceName
+
+    wmInit.clearDatabase(modules)
```

Comment:
params and modules are not really used in clearDabase method. You can just do wmInit.clearDatabase() which will delete all the data. (Unless you want to delete specific modules, in that case WMInit clearDatabase method need to be modified.